.github/ISSUE_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@ This issue tracker is only for technical issues related to bitcoin-core.
 
 General bitcoin questions and/or support requests and are best directed to the [Bitcoin StackExchange](https://bitcoin.stackexchange.com).
 
+For reporting security issues, please use [security@bitcoincore.org](mailto:security@bitcoincore.org).
```

Comment:
"I wanted to do so first, but the URLs are fragile and can change" so are email addresses.
It's not impossible to update this again, it just should be rare.